specification/security/resource-manager/Microsoft.Security/stable/2018-06-01/security.json

```diff
@@ -0,0 +1,275 @@
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/pricings": {
+            "get": {
+                "x-ms-examples": {
+                    "Get pricings on subscription": {
+                        "$ref": "./examples/Pricings/ListPricings_example.json"
+                    }
+                },
+                "tags": ["Pricings"],
+                "description": "Security pricing configurations in the subscription",
+                "operationId": "Pricings_List",
+                "parameters": [{
+                    "$ref": "#/parameters/ApiVersion"
+                }, {
+                    "$ref": "#/parameters/SubscriptionId"
+                }],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/PricingList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            }
+        },        
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/pricings/{pricingName}": {
+            "get": {
+                "x-ms-examples": {
+                    "Get pricings on subscription": {
+                        "$ref": "./examples/Pricings/GetPricingByName_example.json"
+                    }
+                },
+                "tags": ["Pricings"],
+                "description": "Security pricing configuration in the subscription",
+                "operationId": "Pricings_Get",
+                "parameters": [{
+                    "$ref": "#/parameters/ApiVersion"
+                }, {
+                    "$ref": "#/parameters/SubscriptionId"
+                }, {
+                    "$ref": "#/parameters/PricingName"
+                }],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Pricing"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Update pricing on subscription": {
+                        "$ref": "./examples/Pricings/PutPricingByName_example.json"
+                    }
+                },

```

Comment:
Do you need to add x-ms-long-running-operation": true?

Reply:
Current implementation does not expect it.
should it ?

Reply:
The swagger shows the put as a long running operation that returns a 202, in which case it should have a location header that contains an operation url that can be polled for the status of the put, if that is the case then adding x-ms-long-running-operation will cause code to be generated to support this, if the operation is not async then it should return 200 0r 201 (https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#response)